### PR TITLE
Add Hebrew locale

### PR DESCRIPTION
**Pull request by nktfh100**
Hey!
I have added Hebrew locale

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import './styles.css';
 const i18n = require('i18n');
 
 i18n.configure({
-  locales: ['da','de','en','es','fr','it','pt','ru'],
+  locales: ['da','de','en','es','fr','it','pt','ru', "he"],
```
Comment (philfung) [inline src/index.ts:7]: please insert "he" in alphabetical order, after 'fr'

File: src/index.ts [hunk 2]
```diff
@@ -13,7 +13,8 @@ i18n.configure({
     fr: require('./locales/fr.json'),
     it: require('./locales/it.json'),
     pt: require('./locales/pt.json'),
-    ru: require('./locales/ru.json')
+    ru: require('./locales/ru.json'),
+    he: require('./locales/he.json')
```
Comment (philfung) [inline src/index.ts:17]: alphabetical order

File: README.md [hunk 3]
```diff
@@ -82,7 +82,7 @@ This should be a quick drop-in library into your website.
    });
          
     ret = window.AddToHomeScreenInstance.show('en');        // show "add-to-homescreen" instructions to user, or do nothing if already added to homescreen
-                                                            // The only argument is the language to show the messages in (currently only 'de', 'en', 'pt' and 'fr' are available).
+                                                            // The only argument is the language to show the messages in (currently only 'de', 'en', 'pt', 'fr' and 'he' are available).
```
Comment (philfung) [inline README.md:85]: please change to 
'da','de','en','es','fr', 'he','it','pt','ru'

**Review by philfung — CHANGES_REQUESTED** [review]
thanks for adding!  please make following small changes

**Comment by nktfh100** [comment]
Sure!

**Pull request merged**